dockerfiles/theia-endpoint-runtime/Dockerfile

```diff
@@ -20,8 +20,15 @@ COPY *.json /home/theia-endpoint/
 COPY /src /home/theia-endpoint/src
 RUN cd /home/theia-endpoint/ && yarn install
 
+FROM ${BUILD_ORGANIZATION}/${BUILD_PREFIX}-theia-dev:${BUILD_TAG} as eclipse-che-theia-plugin-ext-builder
+
+# Compile
+COPY /eclipse-che-theia-plugin-ext /home/eclipse-che-theia-plugin-ext
+RUN cd /home/eclipse-che-theia-plugin-ext/ && yarn install
```

Comment:
shouldn't we compile everything with the extension at step 21 ?
we may have some dependency injection between them

Reply:
Do you mean to build in separate folders but in the same container?

Reply:
yes or with yarn workspaces (so single build time)

my issue was if at compile time we're missing some dependencies as you're adding the dependency later but for example all inversify stuff will be missed probably.

Reply:
There is no `lib` folder in this case. Should I run `yarn install` in each folder then?